Makefile

```diff
@@ -352,10 +352,15 @@ firewall-install:
 	&& $(MAKE) deploy-shared-service DIR=${MAKEFILE_DIR}/templates/shared_services/firewall/ BUNDLE_TYPE=shared_service
 
 nexus-install:
-	$(MAKE) bundle-build DIR=${MAKEFILE_DIR}/templates/shared_services/sonatype-nexus/ \
-	&& $(MAKE) bundle-publish DIR=${MAKEFILE_DIR}/templates/shared_services/sonatype-nexus/ \
-	&& $(MAKE) bundle-register DIR="${MAKEFILE_DIR}/templates/shared_services/sonatype-nexus" BUNDLE_TYPE=shared_service \
-	&& $(MAKE) deploy-shared-service DIR=${MAKEFILE_DIR}/templates/shared_services/sonatype-nexus/ BUNDLE_TYPE=shared_service
+	$(MAKE) bundle-build DIR=${MAKEFILE_DIR}/templates/shared_services/certs/ \
+	&& $(MAKE) bundle-publish DIR=${MAKEFILE_DIR}/templates/shared_services/certs/ \
+	&& $(MAKE) bundle-register DIR="${MAKEFILE_DIR}/templates/shared_services/certs" BUNDLE_TYPE=shared_service \
+	&& $(MAKE) deploy-shared-service DIR=${MAKEFILE_DIR}/templates/shared_services/certs/ BUNDLE_TYPE=shared_service PROPS='"domain_prefix": "nexus","cert_name": "nexus-ssl"' \
```

Comment:
Is this the right place to store these `PROPS`?
I assume it's reasonable that someone might want to change these. Ideally, they shouldn't have to edit the Makefile to do so.

Reply:
If they were to change these they would also need to modify the deploy-shared-service line for Nexus in the makefile anyway to make sure that the new values are passed via PROPS to override the defaults (the PROPS for the certs service here specify the Nexus defaults). If they were doing that level of customisation they shouldn't be using the makefile command and should be posting to the API manually as the makefile is intended to cover the automated path only.

Reply:
If we were to store the props elsewhere the only place I could think of is as default values in the certs bundle, like we do with the other shared services - but I think as we intend for the certs service to be re-used for other ssl requirements in future that feels like the wrong place